src-docs/src/views/markdown_editor/markdown_format_sink.js

```diff
@@ -146,5 +159,137 @@ Autoconverted link https://github.com/nodeca/pica (enable linkify to see)
 `;
 
 export default () => {
-  return <EuiMarkdownFormat>{markdownContent}</EuiMarkdownFormat>;
+  const textSizeArray = ['xs', 's', 'm', 'relative'];
+
+  const textSizeOptions = textSizeArray.map((name) => {
+    return {
+      value: name,
+      text: name,
+    };
+  });
+
+  const textColorsArray = [
+    'default',
+    'subdued',
+    'success',
+    'accent',
+    'danger',
+    'warning',
+    'ghost',
+    'inherit',
+    'custom',
+  ];
+
+  const textColorsOptions = textColorsArray.map((name) => {
+    return {
+      value: name,
+      text: name,
+    };
+  });
+
+  const [textSize, setTextSize] = useState(textSizeOptions[2].value);
+  const [fontSizeScale, setFontSizeScale] = useState(16);
+  const [textColor, setTextColor] = useState(textColorsOptions[0].value);
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  const onChangeTextSize = (e) => {
+    setTextSize(e.target.value);
+  };
+
+  const onChangeFontSizeScale = (e) => {
+    console.log(fontSizeScale);
+    setFontSizeScale(e.target.value);
+  };
+
+  const onChangeTextColor = (e) => {
+    setTextColor(e.target.value);
+  };
+
+  const panelClasses = classNames({
+    guideDemo__ghostBackground: textColor === 'ghost',
+  });
+
+  const [color, setColor, colorErrors] = useColorPickerState('#c561dc');
+
+  const onButtonClick = () =>
+    setIsPopoverOpen((isPopoverOpen) => !isPopoverOpen);
+  const closePopover = () => setIsPopoverOpen(false);
+
+  const button = (
+    <EuiButtonEmpty
+      onClick={onButtonClick}
+      iconType="controlsVertical"
+      aria-label="Open props customization"
+      color="text"
+      size="s">
+      Customize props
+    </EuiButtonEmpty>
```

Comment:
This isn't really standing out as a control, I'd just make it a regular primary button.
```suggestion
    <EuiButton
      onClick={onButtonClick}
      iconType="controlsVertical"
      size="s">
      Customize props
    </EuiButton>
```